packages/cloudflare/src/cli/build/patches/plugins/content-updater.ts

```diff
@@ -8,11 +8,21 @@ import { readFile } from "node:fs/promises";
 
 import { type OnLoadArgs, type OnLoadOptions, type Plugin, type PluginBuild } from "esbuild";
 
+/**
```

Comment:
This should not be part of the diff, missing a rebase?

Reply:
oh.... oops... sorry let me have a look 😓